src/bosh_azure_cpi/lib/cloud/azure/vms/vm_manager.rb

```diff
@@ -18,10 +18,16 @@ def initialize(azure_config, registry_endpoint, disk_manager, disk_manager2, azu
       @logger = Bosh::Clouds::Config.logger
     end
 
-    def create(instance_id, location, stemcell_id, vm_props, network_configurator, env)
+    def create(bosh_vm_meta, location, vm_props, network_configurator, env)
       # network_configurator contains service principal in azure_config so we must not log it.
-      @logger.info("create(#{instance_id}, #{location}, #{stemcell_id}, #{vm_props}, ..., ...)")
-      resource_group_name = instance_id.resource_group_name
+      @logger.info("create(#{bosh_vm_meta}, #{location}, #{vm_props}, ..., ...)")
```

Comment:
a little misleading... suggest changing BoshVMMeta.to_s to
"<agent_id: #{@agent_id}, stemcell_id: #{@stemcell_id}>"

Reply:
but the vmmeta.to_s can be used in other places, so let's leave it be.